### PR TITLE
Loop Feature Order parameters

### PR DESCRIPTION
**Pull request by zzuutt**
not correct value : alpha-reverse
replace by alpha_reverse

### CODE REVIEW

**Comment by roadster31** [comment]
By replacing this parameter, you're intriducting a backward compatibility break.

Please leave "alpha-reverse", and deprecate it in the documentation. Then add "alpha_reverse".

**Comment by gillesbourgeat** [comment]
@roadster31 :+1:

**Comment by zzuutt** [comment]
OK 
but it is necessary to change the parameter in the features.html file

**Comment by roadster31** [comment]
> but it is necessary to change the parameter in the features.html file

What do you mean ?

**Comment by zzuutt** [comment]
@roadster31 
nothing, i am stupid ! 
I did not reflect

File: core/lib/Thelia/Core/Template/Loop/Feature.php [hunk 1]
```diff
@@ -71,7 +71,7 @@ protected function getArgDefinitions()
             new Argument(
                 'order',
                 new TypeCollection(
-                    new Type\EnumListType(array('id', 'id_reverse', 'alpha', 'alpha-reverse', 'manual', 'manual_reverse'))
+                    new Type\EnumListType(array('id', 'id_reverse', 'alpha', 'alpha-reverse', 'alpha_reverse' , 'manual', 'manual_reverse'))
```
Comment (bibich) [inline core/lib/Thelia/Core/Template/Loop/Feature.php]: Please could you remove the useless space.

Comment (bibich) [inline core/lib/Thelia/Core/Template/Loop/Feature.php]: And could you squash your commit please.

Comment (zzuutt) [inline core/lib/Thelia/Core/Template/Loop/Feature.php]: I have a problem
I don't know Github but @lovenunu gave me these commands
```
$ git checkout master
$ git pull --ff-only upstream master        (error) 
fatal: 'upstream' does not appear to be a git repository
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.

$ git checkout ma-branche
$ git rebase -i master
$ git push -f origin ma-branche
```

**Comment by zzuutt** [comment]
I have a problem
I don't know Github but @lovenunu gave me these commands
```
$ git checkout master
$ git pull --ff-only upstream master        (error) 
fatal: 'upstream' does not appear to be a git repository
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.

$ git checkout ma-branche
$ git rebase -i master
$ git push -f origin ma-branche
```

**Comment by zzuutt** [comment]
Currently the feature page on back use this order parameter

**Comment by bcabarrou** [comment]
I assume you have `origin` (the default git remote name) as your Thelia fork.

You can check your remotes with:
```
git remote -v
```

Try adding the Thelia repository as the `upstream` remote:
```
git remote add upstream https://github.com/thelia/thelia.git
```

**Comment by zzuutt** [comment]
@bcabarrou 
thx

but  i have other pb
```
$ git remote -v
origin  https://github.com/zzuutt/thelia.git (fetch)
origin  https://github.com/zzuutt/thelia.git (push)
thelia  https://github.com/thelia/thelia.git (fetch)
thelia  https://github.com/thelia/thelia.git (push)

zz@PC-zz ~/Documents/GitHub/thelia (patch-17)
$ git checkout master
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.

zz@PC-zz ~/Documents/GitHub/thelia (master)
$ git pull --ff-only thelia master
From https://github.com/thelia/thelia
 * branch            master     -> FETCH_HEAD
fatal: Not possible to fast-forward, aborting.

zz@PC-zz ~/Documents/GitHub/thelia (master)
$ git remote add upstream https://github.com/thelia/thelia.git

zz@PC-zz ~/Documents/GitHub/thelia (master)
$ git pull --ff-only upstream master
From https://github.com/thelia/thelia
 * branch            master     -> FETCH_HEAD
 * [new branch]      master     -> upstream/master
fatal: Not possible to fast-forward, aborting.
```

**Comment by bcabarrou** [comment]
Sounds like your local `master` branch has changes not in the Thelia `master` branch.

**Comment by gillesbourgeat** [comment]
@zzuutt Your commit was cherry pick in this pull request #1784

**Comment by zzuutt** [comment]
@gillesbourgeat 
thx
Seen the problems which I meet with my branch
I'll delete it

**Pull request closed without merging**